ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -411,12 +411,17 @@ def __lt__(self, value, /):
     def __int__(
         self,
     ):
-        return ivy.array(ivy.reshape(self._ivy_array, -1), dtype=ivy.int64)[0]
+        return ivy.array(ivy.reshape(self._ivy_array, (-1,)), dtype=ivy.int64)[0]
```

Comment:
Needed because it would bomb in other frameworks such as in 

> ================================================================================================================== short test summary info ==================================================================================================================
> FAILED ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py::test_numpy_instance_float__[cpu-ivy.functional.backends.torch-False-False] - ivy.utils.exceptions.IvyBackendException: torch: reshape: reshape(): argument 'shape' (position 2) must be tuple of ints, not int
> FAILED ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py::test_numpy_instance_float__[cpu-ivy.functional.backends.paddle-False-False] - ivy.utils.exceptions.IvyBackendException: paddle: reshape: object of type 'int' has no len()

Reply:
I see.